### PR TITLE
NTR - unify article message string and add missing ordernumber for de…

### PR DESCRIPTION
**Pull request by teiling88**
…tail inquiry message

### 6. Checklist

- [ ] I have written tests and verified that they fail without my change
- [ ] I have squashed any insignificant commits
- [ ] This change has comments for package types, values, functions, and non-obvious lines of code
- [ ] I have read the contribution requirements and fulfil them.

### CODE REVIEW

File: engine/Shopware/Controllers/Frontend/Forms.php [hunk 1]
```diff
@@ -242,7 +242,7 @@ protected function getContent($formId)
                             if ($this->Request()->getParam('sOrdernumber') !== null) {
                                 $getName = Shopware()->Modules()->Articles()->sGetArticleNameByOrderNumber($this->Request()->getParam('sOrdernumber'));
                                 $text = Shopware()->Snippets()->getNamespace('frontend/detail/comment')->get('InquiryTextArticle');
-                                $text .= ' ' . $getName;
+                                $text .= ' ' . $getName . ' (' . $this->Request()->getParam('sOrdernumber') . ')';
```
Comment (shyim) [inline engine/Shopware/Controllers/Frontend/Forms.php]:
```suggestion
                                $text .= ' ' . $getName . ' (' . htmlentities($this->Request()->getParam('sOrdernumber')) . ')';
```

Secure is secure

Comment (teiling88) [reply engine/Shopware/Controllers/Frontend/Forms.php]: Done

File: engine/Shopware/Controllers/Frontend/Forms.php [hunk 2]
```diff
@@ -242,7 +242,7 @@ protected function getContent($formId)
                             if ($this->Request()->getParam('sOrdernumber') !== null) {
                                 $getName = Shopware()->Modules()->Articles()->sGetArticleNameByOrderNumber($this->Request()->getParam('sOrdernumber'));
                                 $text = Shopware()->Snippets()->getNamespace('frontend/detail/comment')->get('InquiryTextArticle');
-                                $text .= ' ' . $getName;
+                                $text .= ' ' . $getName . ' (' . htmlentities($this->Request()->getParam('sOrdernumber'), ENT_QUOTES | ENT_HTML5) . ')';
```
Comment (mitelg) [inline engine/Shopware/Controllers/Frontend/Forms.php]: `$this->Request()->getParam('sOrdernumber')` is now called three times. we should use a variable :wink:

Comment (teiling88) [reply engine/Shopware/Controllers/Frontend/Forms.php]: Fixed

**Comment by shyim** [comment]
Thanks for your Contribution! 💙

**Pull request closed without merging**